src/sentry/static/sentry/app/views/settings/account/accountSecurity/accountSecurityDetails.jsx

```diff
@@ -57,6 +58,10 @@ class AuthenticatorDate extends React.Component {
 }
 
 class AccountSecurityDetails extends AsyncView {
+  static PropTypes = {
+    deleteDisabled: PropTypes.bool.isRequired,
+    handleRegenerateBackupCodes: PropTypes.func.isRequired,
```

Comment:
nit: use `on`-prefix when defining a component callback (to be consistent with react's `on<Event>`s and `handle`-  for the method/function that will need to use the callback.